### ISSUE TITLE
globals use char error

### ISSUE THREAD

**Issue by widuu** [zephir-lang/zephir#1035]
When I added a globals type of char to the wrong

`config.json`

```
"microdb_dsn": {
            "type": "char",
            "default": "",
            "ini-entry": {
                "name": "microdb.dsn",
                "scope": "PHP_INI_ALL"
            }
        }
```

Compiled c file 

```
microdb_globals->microdb_dsn = ''";
```

So there is an error ！ And comfile-errors.log 

```
/Users/widuu/git/microdb/ext/microdb.c:102:33: warning: empty character constant [-Winvalid-pp-token]
        microdb_globals->microdb_dsn = ''";
                                       ^
/Users/widuu/git/microdb/ext/microdb.c:102:33: error: expected expression
/Users/widuu/git/microdb/ext/microdb.c:102:35: warning: missing terminating '"' character [-Winvalid-pp-token]
        microdb_globals->microdb_dsn = ''";
                                         ^
2 warnings and 1 error generated.
make: *** [microdb.lo] Error 1
make: *** Waiting for unfinished jobs....
/Users/widuu/git/microdb/ext/microdb.c:102:33: warning: empty character constant [-Winvalid-pp-token]
        microdb_globals->microdb_dsn = ''";
                                       ^
/Users/widuu/git/microdb/ext/microdb.c:102:33: error: expected expression
/Users/widuu/git/microdb/ext/microdb.c:102:35: warning: missing terminating '"' character [-Winvalid-pp-token]
        microdb_globals->microdb_dsn = ''";
                                         ^
2 warnings and 1 error generated.
make: *** [microdb.lo] Error 1

```

**Comment by Jeckerson** [comment]
Related https://github.com/zephir-lang/zephir/issues/2280

**Comment by Jeckerson** [comment]
Use `string` type instead. Available since version `0.15.1`. 
Closing.
